Gems/Terrain/Code/Source/TerrainRenderer/TerrainFeatureProcessor.h

```diff
@@ -122,24 +118,35 @@ namespace Terrain
         AZ::RHI::Ptr<AZ::RHI::Buffer> m_indexBuffer;
         AZ::RHI::Ptr<AZ::RHI::Buffer> m_vertexBuffer;
         AZ::RHI::IndexBufferView m_indexBufferView;
-        AZStd::fixed_vector<AZ::RHI::StreamBufferView, AZ::RHI::Limits::Pipeline::StreamCountMax> m_vertexBufferViews;
+        AZ::RHI::StreamBufferView m_vertexBufferView;
 
         // Per-area data
         struct TerrainAreaData
         {
             AZ::Transform m_transform;
-            AZ::Aabb m_terrainBounds;
+            AZ::Aabb m_terrainBounds{ AZ::Aabb::CreateNull() };
             float m_heightScale;
             AZ::Data::Instance<AZ::RPI::StreamingImage> m_heightmapImage;
             uint32_t m_heightmapImageWidth;
             uint32_t m_heightmapImageHeight;
             bool m_propertiesDirty{ true };
```

Comment:
Should any of these other members be initialized?

Reply:
I wanted to be explicit for the bounds because I'm using it's null state to check the validity of the struct. The only other member that doesn't have an obvious default state is the Transform, so I could see a case for initializing that.